webapp/components/PostCard/PostCard.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <ds-card
+    :lang="post.language"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-14T23:50:49Z" title="Friday, November 15th 2019, 12:50:49 am +01:00">Nov 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,5 +1,6 @@
 <template>
   <ds-card
+    :lang="language"
```


```suggestion
    :lang="post.language"
```

Reply:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2019-11-15T20:21:01Z" title="Friday, November 15th 2019, 9:21:01 pm +01:00">Nov 15, 2019</time>_

---


I can follow your argumentation, and I would prefer your solution if there were not those 18k posts without language. This would lead to something like `lang="null"` and I am not sure if this is what we want. I think it is up to the team to decide which is the better way to handle this problem.